### ISSUE TITLE
How to get the SSH root keys `step ca root --ssh` or `step ca ssh root` or `step ssh root`...

### ISSUE THREAD

**Issue by CMCDragonkai** [smallstep/cli#206]
### What would you like to be added

There's a command called `step ca root` which gets the root PKI cert.

But we also need similar commands for the SSH root keys as well. There is currently 2 `ssh_host_key.pub` and `ssh_user_key.pub`. The former used to sign host keys, the latter used to sign user keys. 

I would like for there to be a similar command to fetch these 2 "root keys".

### Why this is needed

When using SSH CA, we need to add these lines:

```
TrustedUserCAKeys /etc/ssh/ssh_user_key.pub
```

So that hosts can trust user keys that are signed by the CA.

On the other hand, for users to trust host keys signed by the CA, they need to add this:

```
@cert-authority *.example.com ecdsa-sha2-nistp256 AAAAE...=
```

So in both cases we need the user root key and the host root key.

Also it would be cool, if the step server exposed a RESTful API for acquiring these root keys. Not every "client-side" has the `step` client. These keys are meant to be public, so why not just expose them directly via HTTPs.

**Comment by maraino** [comment]
In the master branch we have:
* `step ssh config --roots` to get the public key used to sign user certificates.
* `step ssh config --roots --host` to get the public key used to sign host certificates.

We might change these commands before releasing, see https://github.com/smallstep/cli/issues/212

And `step ssh config` / `step ssh config --host` now can automatically configure user and host certificates using the templates generated by `step ca init --ssh`

**Comment by CMCDragonkai** [comment]
What does this mean?

> And step ssh config / step ssh config --host now can automatically configure user and host certificates using the templates generated by step ca init --ssh

**Comment by maraino** [comment]
@CMCDragonkai Those commands are available in the upcoming 0.14 release of `step`, and `step-ca`, they are currently available in builds from the `master` branch.

On 0.14, if `step-ca` is initialized with `step ca init --ssh`, it will have some templates that can be used to automatically configure an ssh client or an ssh server to use ssh certificates signed by `step-ca`.

The command `step ssh config` will configure the client, and `step ssh config --host` will configure a server. The default templates are available [here](https://github.com/smallstep/certificates/blob/master/pki/templates.go). And for example, a client will be configured with one include in ~/.ssh/config that will read ~/.step/ssh/config, the default for templates for these actions are:
```
Host *
{{- if or .User.GOOS "none" | eq "windows" }}
	Include "{{ .User.StepPath | replace "\\" "/" | trimPrefix "C:" }}/ssh/config"
{{- else }}
	Include "{{.User.StepPath}}/ssh/config"
{{- end }}
```
```
Match exec "step ssh check-host %h"
	ForwardAgent yes
{{- if .User.User }}
	User {{.User.User}}
{{- end }}
{{- if or .User.GOOS "none" | eq "windows" }}
	UserKnownHostsFile "{{.User.StepPath}}\ssh\known_hosts"
	ProxyCommand C:\Windows\System32\cmd.exe /c step ssh proxycommand %r %h %p
{{- else }}
	UserKnownHostsFile "{{.User.StepPath}}/ssh/known_hosts"
	ProxyCommand step ssh proxycommand %r %h %p
{{- end }}
```

This last template does a request to `step-ca` to see if the host has a signed certificate, if it does, it will use a custom known_host file that will contain the key used to verify the server certificates, and it will use a custom proxy command that will just proxy the connection to `ssh` or use `ssh` to proxy the connection if a bastion is configured in `step-ca`.
